assets/css/detours/_detour_table.scss

```
@@ -1,8 +1,65 @@
+@use "../color/tokens_2024" as tokens;
+
+$table-border: 1px solid tokens.$gray-500;
```

The `$gray-500` was looking a little intense to my eye, and checking the [figma](https://www.figma.com/design/iQiQA1toiMfPDhSbTYg2cx/Skate-%7C-Unplanned-Detours-%7C-Exploration?node-id=1783-2059&t=qnIFHo7vhooGGzaH-4), it seems the border should a bit more delicate: `#00000020`. (I'm not worried about making it a reusable color.)
```suggestion
$table-border: 1px solid #00000020;
```

---

Actually, the figma I attached was not the final handed off version. I cannot find the handoff doc

---

Ok yep! This is different in the official [hand-off](https://www.figma.com/design/iQiQA1toiMfPDhSbTYg2cx/Skate-%7C-Unplanned-Detours-%7C-Exploration?node-id=1956-2816&t=vAH4efrkwgE3UVA6-4). (How interesting!)

Sadly, the handoff uses 2021 colors 🥲  I'd rather keep this file relying on the 2024 colors, though. It doesn't seem worthwhile using a deprecated color for a border, of all things.

```suggestion
$table-border: 1px solid tokens.$gray-400;
```

---

Fascinating - not sure how I goofed that